pkg/operator/controller/ingress/controller.go

```diff
@@ -152,7 +164,13 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	// successful, immediately re-queue to refresh state.
 	if !isAdmitted(ingress) {
 		if err := r.admit(ingress, ingressConfig, infraConfig); err != nil {
-			return reconcile.Result{}, fmt.Errorf("failed to admit ingresscontroller: %v", err)
+			switch err := err.(type) {
+			case *admissionRejection:
+				log.Info("rejected ingresscontroller", "ingresscontroller", ingress, "reason", err.Error())
```

Comment:
In this context `err` is of type `admissionRejection` and so you can reference `err.Reason`

Reply:
Done.